code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -31,6 +31,20 @@
 	no_equip = list(ITEM_SLOT_BACK)
 	mutanttongue = /obj/item/organ/tongue/kepori
 	species_language_holder = /datum/language_holder/kepori
+	/// For storing our tackler datum so we can remove it after
+	var/datum/component/tackler
```

Comment:
we usually dont want to store components on anything, weakrefs would be better but still eh, you could also just qdel(C.GetComponent(/datum/component/tackler))